crates/sage-cli/src/output.rs

```diff
@@ -145,6 +176,41 @@ impl Runner {
         Ok(path.to_string())
     }
 
+    pub fn write_fragments( &self,features: &[Feature]) -> anyhow::Result<String>{
```

Comment:
Run `cargo fmt`

Reply:
I have implemented the necessary formatting to the code.